README.md

```diff
@@ -33,6 +33,9 @@ More information can be found at the links below:
 - `services.startNotificationProxyService`
 - `services.startHouseArrestService`
 
+### Environment
+- `USBMUXD_SOCKET_ADDRESS` follow [libusbmuxd](https://github.com/libimobiledevice/libusbmuxd) to get usbmuxd socket address from USBMUXD_SOCKET_ADDRESS environment.
```

Comment:
Please describe how an example/acceptable value should look like and a purpose of it